octopusdeploy_framework/schemas/community_step_template.go

```diff
@@ -262,52 +230,37 @@ func GetReadOnlyStepTemplateParameters() rs.ListNestedAttribute {
 				"default_value": util.ResourceString().
 					Description("A default value for the parameter, if applicable. This can be a hard-coded value or a variable reference.").
 					Computed().
-					Default(stringdefault.StaticString("")).
 					PlanModifiers(stringplanmodifier.UseStateForUnknown()).
 					Build(),
 				"default_sensitive_value": util.ResourceString().
 					Description("Use this attribute to set a sensitive default value for the parameter when display settings are set to 'Sensitive'").
-					Optional().
+					Computed().
```

Comment:
This was also a mistake in the previous PR where the value should have been computed.